101-private-dns-zone/README.md

```diff
@@ -0,0 +1,16 @@
+# Azure private DNS zone
```

Comment:
check the pipeline results for fixes needed here

Reply:
Readme is missing badge: https://azurequickstartsservice.blob.core.windows.net/badges/101-private-dns-zone/PublicLastTestDate.svg

This is not in the instructions of the contribution sample readme. I do not have access to this container. Is this dynamically generated once I add the url?

Reply:
@bmoore-msft ok, figured it out. Looks like the build passed now.